Makefile

```diff
@@ -17,7 +17,7 @@ STRING_SET_VALUES := --set namespace=$(NAMESPACE),versionUpgradeHook.name=$(UPGR
 
 DOCKERFILE ?= operator
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:crdVersions=v1"
+CRD_OPTIONS ?= "crd:crdVersions=v1,allowDangerousTypes=true"
```

Comment:
since we use float64 (which is the base type of the automation setting), we will need to use those dangerousTypes.

Alternative:
- use ints
- larger refactoring for support of quantity/strings (not preferred)